app/src/router/index.ts

```
@@ -136,6 +166,12 @@ router.afterEach((to) => {
     routeMemoryStore.setLastCommunityRoute(communityId, to.path, to.params);
     if (channelId && viewId) routeMemoryStore.setLastChannelView(communityId, channelId, viewId);
   }
+
+  // Open call window when navigating to a channel from outside the app (e.g., shared link)
+  if (to.name === 'channel' && !from.name) {
+    const uiStore = useUiStore();
```

Biome (2.3.13)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)